Dockerfile

```diff
@@ -18,7 +18,7 @@ ARG SWUPD_UPDATE_ARG
 ARG NDCTL_VERSION="66"
 ARG NDCTL_CONFIGFLAGS="--disable-docs --without-systemd --without-bash"
 ARG NDCTL_BUILD_DEPS="os-core-dev devpkg-util-linux devpkg-kmod devpkg-json-c"
-ARG GO_VERSION="1.12.9"
+ARG GO_VERSION="1.13"
```

Comment:
Isn't this unrelated to this PR?

Reply:
Never mind. Now I remember that you wanted to use Go 1.13 for error handling. Please call that out in the PR subject and description, because it is a major change by itself.

Reply:
Actually, it probably would be better to do the Go update in a separate PR.

Reply:
Now this change is in separate PR #461